apps/api/prisma/migrations/20230218120531_discord_active_added/migration.sql

```diff
@@ -0,0 +1,16 @@
+/*
+  Warnings:
+
+  - The `pin` column on the `User` table would be dropped and recreated. This will lead to data loss if there is data in the column.
+
+*/
+-- AlterTable
+ALTER TABLE "College" ALTER COLUMN "id" SET DEFAULT gen_random_uuid()::TEXT;
+
+-- AlterTable
+ALTER TABLE "Skill" ALTER COLUMN "id" SET DEFAULT gen_random_uuid()::TEXT;
+
+-- AlterTable
+ALTER TABLE "User" ADD COLUMN     "discordActive" BOOLEAN NOT NULL DEFAULT false,
+DROP COLUMN "pin",
```

Comment:
I cannot merge this. Change this to Alter and not drop as prisma says this will cause data loss